alembic/versions/33de75e55858_adding_datetime_column.py

```diff
@@ -0,0 +1,44 @@
+"""Adding datetime column
+
+Revision ID: 33de75e55858
+Revises: 525b125ec799
+Create Date: 2015-10-10 12:32:39.936107
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '33de75e55858'
+down_revision = '525b125ec799'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('campaign_anon_view', sa.Column('datetime', sa.DateTime()))
+    op.create_index(op.f('ix_campaign_anon_view_datetime'), 'campaign_anon_view', ['datetime'], unique=False)
+    op.execute('UPDATE "campaign_anon_view" set datetime = created_at')
```

Comment:
Using a raw SQL statement is a cheap shot that works because this particular syntax works with PostgreSQL, our current database. To be backend-safe, you should let SQLAlchemy generate SQL for you, like this: https://github.com/hasgeek/hasjob/blob/master/alembic/versions/441fd49059e3_preserve_user_s_name.py#L29